docs/reaction_diffusion_solver.rst

```diff
@@ -1,7 +1,137 @@
+* **Element** ``<DiffusionField>``
+    * Defines a diffusion field
+    * **Attribute** ``Name``: the name of the field
+    * **Element** ``<DiffusionData>``
+        * **Element** ``<AdditionalTerm>`` (optional)
+            * Contains a `muParser expression <mu_parser.html>`_ that determines the reaction diffusion equation. See :ref:`here <AdditionalTerm>` for more details
+            * The expression can be unique for each CellType with the use of ternary operators. 
+        * **Element** ``<DecayCoefficient>`` (optional)
+            * Specifies a constant decay coefficient for a cell type.
+            * Can be set per cell during simulation execution
+            * **Attribute** ``CellType``: name of the cell type
+        * **Element** ``<DecayConstant>`` (optional)
+            * Specifies a constant diffusion coefficient for the medium.
+            * Note that ``<GlobalDecayConstant>`` performs the same function, despite its name.
+        * **Element** ``<DiffusionCoefficient>`` (optional)
+            * Specifies a constant diffusion coefficient for a cell type.
+            * Can be set per cell during simulation execution
+            * **Attribute** ``CellType``: name of the cell type
+        * **Element** ``<DiffusionConstant>`` (optional)
+            * Specifies a constant diffusion coefficient for the medium.
+            * Note that ``<GlobalDiffusionConstant>`` performs the same function, despite its name.
+    * **Element** ``<SecretionData>`` (optional)
+        * Secretion data elements, defined in the same way as for :ref:`FlexibleDiffusionSolverFE <SecretionData>`
+    * **Element** ``<BoundaryConditions>`` (optional)
+        * Boundary condition elements, defined in the same as for DiffusionSolverFE.
+        * Boundary conditions are applied at surfaces and can be manipulated at each site during simulation execution.
+        * If a condition is not specified for a boundary, then it is assumed to be zero flux.
+        * `BoundaryConditions <boundary_conditions_diffusion.html>`_ options:
+            * NoFlux (default)
+            * ConstantValue
+            * ConstantDerivative
+            * Periodic
+* **Element** ``<ExtraTimesPerMCS>`` (optional)
+    * Allows calling of diffusion (and only diffusion) more than once per MCS
+    * It is intended to be used with ``<DeltaT>`` and ``<DeltaX>``
+    * See :ref:`DeltaT <DeltaT>` and `Diffusion Solver Settings <diffusion_solver_settings.html>`_ for more details
+* **Element** ``<FluctuationCompensator>`` (optional)
+    * Enables deployment of the CC3D FluctuationCompensator.
+* **Element** ``<InitialConcentrationExpression>`` (optional)
+    * Contains a `muParser expression <mu_parser.html>`_ to define the level of concentration at each voxel on MCS 0.
+    * We recommend using either InitialConcentrationExpression or ConcentrationFileName but not both.
```

Comment:
I could not confirm that `InitialConcentrationExpression` is a part of ReactionDiffusionSolverFE. It seems to be present in all of the other PDE solvers, however.